src/Adapter/Presenter/Cart/CartPresenter.php

```diff
@@ -89,7 +90,9 @@ public function __construct()
         $this->priceFormatter = new PriceFormatter();
         $this->link = $context->link;
         $this->translator = $context->getTranslator();
-        $this->imageRetriever = new ImageRetriever($this->link);
+
+        $isAvifAvailable = new AvifExtensionChecker(Configuration::get('PS_ADDITIONAL_IMAGE_AVIF'));
+        $this->imageRetriever = new ImageRetriever($this->link, $isAvifAvailable);
```

Comment:
```suggestion
        $isAvifAvailable = new AvifExtensionChecker(Configuration::get('PS_ADDITIONAL_IMAGE_AVIF'));
        $this->imageRetriever = new ImageRetriever($this->link, $isAvifAvailable);
```

I don't like this, I really don't. I think it's worth thinking of a different way of checking if `Avif` is there, otherwise, we double the amount of the code we need to get the ImageRetriever. It's not a Symfony service, maybe we should initialize it directly in the ImageRetriever. In the future, after migrating this to Symfony, it'd be simpler to handle having this in a constructor.